includes/core.php

```diff
@@ -111,8 +91,11 @@ function notify_user_of_missing_requirements() {
 ?>
 
 	<div class="notice notice-warning">
-		<p><?php esc_html_e( 'The Airstory plugin is missing one or more of its dependencies, so it\'s automatically been deactivated.', 'airstory' ); ?></p>
-		<p><?php echo wp_kses_post( __( 'For more information, <a href="#" target="_blank">please see the plugin\'s <abbr title="Frequently Asked Questions">FAQ</abbr></a>.', 'airstory' ) );?></p>
+		<p><?php esc_html_e( 'The Airstory plugin is missing one or more of its dependencies, so it\'s not yet available to users on this site.', 'airstory' ); ?></p>
+		<p><?php echo wp_kses_post( sprintf(
```

Comment:
I typically try to avoid combining `sprintf()` and string concatentation, as the former is a more elegant replacement for the latter. Placeholders also become more difficult for plugin translators, as it's kind of a "well, let's drop in opening/closing tags as placeholders and hope for the best", which feels kinda kludgy to me.

A couple years ago, a colleague also turned me onto [Zach Tollman's performance audit on the KSES series of functions](https://www.tollmanz.com/wp-kses-performance/), where he comes to the conclusion that KSES, when applied selectively, are not a significant hinderance to performance (especially when we're running it on small, short strings).